src/Pages/SignIn/SignIn.scss

```
@@ -1,2 +1,211 @@
 @import "../../Styles/common.scss";
-@import "../../Styles/reset.scss";
+
+.SignIn {
+  padding: 60px 0;
+  .wrapper {
+    display: flex;
+    justify-content: center;
+    padding: 50px 0 80px 0;
+    .wrapperLeft {
+      flex-direction: column;
+      justify-content: center;
+      margin-top: 50px;
+      .infoBanner {
+        .bannerFontTop {
+          font-size: 25px;
+          font-weight: 400;
+        }
+        .bannerFontBot {
+          font-size: 13px;
+          color: #7c7c7c;
+        }
+      }
+      .bannerImg img {
+        width: 540px;
+        height: 300px;
+      }
+    }
+    .wrapperRight {
+      padding: 60px 70px;
+      border: 1px solid #e5e5e5;
+      .infoLogo {
+        margin: 0 0 30px 0;
+        .infoServiceLogo {
+          width: 100px;
+          height: 30px;
+        }
+      }
+      .loginForm {
+        .loginTop {
+          display: flex;
+          flex-direction: column;
+          .idPwForm {
```

```suggestion
        .loginTop {
          display: flex;
          flex-direction: column;
          
          .idPwForm {
```
- 새로운 셀렉터가 시작 될 때 한줄 띄워주세요
- flex -> column은 block과 같습니다. flex가 필요한 상황인지 확인 후 사용해주세요!

---

수정완료했습니다 !